src/index.js

```diff
@@ -329,17 +351,23 @@ const reduceRules = (rules, attributes) => {
  *
  * @param {object} policy the policy to evaluate
  * @param {object} attributes the attributes to use for the evaluation
+ * @param {object} options optional function config
+ * @param {array} options.inlineTargets optional list of attribute paths that
+ * should be eagerly evaluated when reducing the policy. Eager evaluation makes
+ * sure that a rule with a known target will be inverted and replaced with the
+ * known value in-line.
  * @returns {object} the policy reduced to conditions involving attributes not
  * not given
  * @throws {Error} if the policy is invalid
  */
-const reduce = (policy, attributes) => {
-  const result = {};
-
+const reduce = (policy, attributes, options = {}) => {
   validate(policy);
+  validateJsonSchema('ReduceOptions', options);
```

Comment:
I think this is a breaking change unless we do this, right?
```suggestion
  if (options) {
    validateJsonSchema('ReduceOptions', options);
  }
```

Reply:
@swain it's not. `reduce` defaults options to `{}`, and the `inlineTargets` prop is optional in the schema.